app/controllers/reviews_controller.rb

```diff
@@ -18,6 +18,12 @@ def index
     render json: @reviews
   end
 
+  def destroy
+    @review.destroy
+    @reviews = @room.reviews
```

Comment:
why are you returning all of the reviews here again? I think you just want destroy to return a success status of :201. You already have the reviews in the DOM, so this would be a waste of network bandwidth on your server.